app/services/decisions/capital_decision_tree.rb

```diff
@@ -48,6 +55,18 @@ def after_savings_summary
       edit(:other_saving_type)
     end
 
+    def after_investment_type(investment)
+      return edit(:premium_bonds) unless investment
```

Comment:
```
# TODO: Route to trust fund question once built
edit('/steps/case/urn') # Placeholder to join up flow
```

Reply:
Only bc I got caught in a loop in local and got confused!

Reply:
I've fixed this in the next PR.